### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by guotie**
use pino-elasticsearch as module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,35 @@ npm install pino-elasticsearch -g
 You can then use [Kibana](https://www.elastic.co/products/kibana) to
 browse and visualize your logs.
 
+## Used as a module
+
+use pino-elasticsearch as a module is sample, use pino-multi-stream to send log to multi transport, for example:
```
Comment (mcollina) [inline README.md]:
```suggestion
use pino-elasticsearch as a module is simple, use [pino-multi-stream](https://www.npmjs.com/package/pino-multi-stream) to send log to multi transport, for example:
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,35 @@ npm install pino-elasticsearch -g
 You can then use [Kibana](https://www.elastic.co/products/kibana) to
 browse and visualize your logs.
 
+## Used as a module
```
Comment (mcollina) [inline README.md]:
```suggestion
## Use as a module
```

File: README.md [hunk 3]
```diff
@@ -38,6 +38,35 @@ npm install pino-elasticsearch -g
...
+
+use pino-elasticsearch as a module is sample, use pino-multi-stream to send log to multi transport, for example:
+
+```
```
Comment (mcollina) [inline README.md]:
```suggestion
```js
```

File: README.md [hunk 4]
```diff
@@ -38,6 +38,35 @@ npm install pino-elasticsearch -g
...
+  })
+
+
+logger.info({'msg': {'info': 'info'}})
+logger.debug('debug')
+logger.warn('warn')
+logger.error('error')
+
+```
+*** Notice, create the pino-elasticsearch must set host & port ***
```
Comment (mcollina) [inline README.md]:
```suggestion

*** Notice, the `host` and `port` parameters of `pino-elasticsearch` are required ***
```

**Comment by mcollina** [comment]
@guotie I don't understand why you resolved all my notes without actually landing the proposed changes. Would you mind accepting them so we can land this?

**Comment by guotie** [comment]
accept.

**Comment by mcollina** [comment]
I cannot land those updates as I do not have write permissions on your branch. You should accept those.

**Pull request merged**